setup.cfg

```diff
@@ -13,6 +13,7 @@ classifiers =
 [options]
 packages = find:
 install_requires =
+    --future--
```

Comment:
```suggestion
```

`__future__` is built in so no need to add to the install list here.

Reply:
looks like this is maybe also an invalid package name (starting with a hyphen i think?) which caused the build to fail:

https://readthedocs.org/projects/mpl-point-clicker/builds/23301508/